recipe/deploy/cleanup.php

```diff
@@ -18,6 +18,7 @@
     }
 
     foreach ($releases as $release) {
+        run("ionice -c 3 nice -19 find {{deploy_path}}/releases/$release/ -delete", $runOpts);
```

Comment:
I think this line deserves a comment, what it is expected todo.
never heard of this command.

Reply:
Will it work on all *nix systems? Solaris? Bsd?

Reply:
Not sure if this is a solution for general case. Maybe your setup is too specific.

Reply:
Yep, I should delete ionice, Bsd doesn't support.